Library/ViewModels/PledgeViewModelTests.swift

```diff
@@ -999,8 +999,16 @@ final class PledgeViewModelTests: TestCase {
     )
   }
 
-  func testGoToThanks() {
-    withEnvironment(apiService: MockService()) {
+  func testApplePay_GoToThanks() {
```

Comment:
In order to kind've prove that there were no regressions I left these tests largely as-is. But we can chat about potentially adding more? I just didn't want this PR to be 1000+ lines from the start 😅